go.mod

```diff
@@ -21,13 +21,15 @@ require (
 	github.com/containernetworking/plugins v1.1.1
 	github.com/coreos/go-iptables v0.8.0
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
+	github.com/emanic/gaia v14.272.0+incompatible
```

Comment:
The filed type was changed to int-or-string during the API review. But due the recent changes, it would be better to restrict it as `string` and use `enum` (tcp, udp, icmp). 

updated.